vendor/Makefile.am

```diff
@@ -0,0 +1,9 @@
+AUTOMAKE_OPTIONS = subdir-objects
+
+libboost_%.built:
+	$(SHELL) -c "cd $(srcdir)/boost; ./b2 --build-dir=build --prefix=install --with-$* link=static $(BOOST_TOOLSET) install"
+	touch libboost_$*.built
+
+clean-local:
+	$(SHELL) -c "cd $(srcdir)/boost; git clean -df"
```

Comment:
The `-df` is actually on purpose here because it doesn't remove the `b2` utility which is created as part of `./autogen.sh`. `make clean` should only remove the the artifacts from `make [SOMETHING]`. So the `-xdf` variant should go into `distclean-local`.